### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by HanaSabih**
Saved the ID of my game that will be returned by API.
Implement the "Refresh" button that will get all scores for a game created by me from the API 
Implement the form "Submit" button that will save a score for a game created by me(sending data to the API).
Use arrow functions instead of the function keyword.
Use async and await JavaScript features to consume the API.

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @HanaSabih, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - Unfortunately, you submitted the project 'Hit the API' on the 'Final touches' project submission. Please use the correct link for submitting a project and double-check where you are submitting it.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

File: src/modules/variables.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export const form = document.querySelector('#form');
+export const warningLabel = document.querySelector('.warningLabel');
+export const refreshScores = document.querySelector('#refreshScores');
+export const scoreInput = document.querySelector('#scoreInput');
+export const nameInput = document.querySelector('#nameInput');
```
Comment (OLIPLICHE) [inline src/modules/variables.js:5]: # Optional:
- I would advise you to keep the number of changes/updates to the DOM as low as possible, they are very expensive for the browser.

File: src/index.html [hunk 2]
```diff
@@ -23,19 +23,16 @@ <h1 class="p-5">Leaderboard</h1>
...
-            <tbody class="tableBody">
-              <tr>
-                <td>Name : 123456</td>
-              </tr>
-              <tr>
-                <td>Name : 123456</td>
-              </tr>
-            </tbody>
+            <tbody class="tableBody"></tbody>
           </table>
```
Comment (OLIPLICHE) [inline src/index.html:36]: - Okay 👍 
Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi <STUDENT>, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**